src/main/java/pl/javastart/task/ControlStation.java

```diff
@@ -0,0 +1,28 @@
+package pl.javastart.task;
+
+import java.util.Queue;
+
+public class ControlStation {
+    private Queue<Vehicle> vehicles;
+    private String fileName;
+    private FileOperator operator = new FileOperator();
+
+    public ControlStation(String fileName) {
+        this.fileName = fileName;
+        vehicles = operator.readVehicles(fileName);
+    }
+
+    public void addVehicle(Vehicle vehicle) {
+        vehicles.offer(vehicle);
+    }
+
+    public Vehicle readNextVehicle() {
+        return vehicles.poll();
+    }
+
+    public void saveVehicles() {
+        if (vehicles.size() > 0) {
+            operator.saveVehicles(vehicles, fileName);
+        }
```

Comment:
Jeżeli kolejka jest pusta, to też chcemy zapisać pusty plik i go de facto wyczyścić.
Jeżeli w pliku były 3 pojazdy przed uruchomieniem aplikacji, obsłużymy wszystkie i zamkniemy aplikację, to z aktualną logiką przy kolejnym uruchomieniu aplikacji będziemy obsługiwali te same pojazdy po raz drugi